relation_engine_server/test/data/json_validation/invalid_date.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "invalid_date",
+  "distance": 1,
+  "date": "20200606"
+}
```

Comment:
This is nitpicky, don't need to change this here, but some of these files have newlines at the end and some dont. It's best to have newlines at the end of all text files so that it works well with shell tools (eg https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
(usually the editor should insert a newline at the end of the file, without any blank line visible to the user)

Reply:
Done!